crypto/rand/rand_lcl.h

```diff
@@ -203,7 +209,17 @@ struct rand_drbg_st {
 /* The global RAND method, and the global buffer and DRBG instance. */
 extern RAND_METHOD rand_meth;
 
-/* How often we've forked (only incremented in child). */
+/*
+ * A "generation count" of forks.  Incremented in the child process after a
+ * fork.  Since rand_fork_count is increment-only, and only ever written to in
+ * the child process of the fork, which is guaranteed to be single-threaded, no
+ * locking is needed for normal (read) accesses; the rest of fork processing is
```

Comment:
What do you think about [my suggestion](https://github.com/openssl/openssl/pull/4110#discussion_r172009849) to write "pthread fork processing" instead of "fork processing" to please @richsalz (favoured also by @levitte)?

Reply:
That is a good change; sorry for missing it in my hasty review of the outstanding comments.